openadapt/crud.py

```diff
@@ -67,7 +97,16 @@ def insert_screenshot(recording_timestamp, event_timestamp, event_data):
     _insert(event_data, Screenshot, screenshots)
 
 
-def insert_window_event(recording_timestamp, event_timestamp, event_data):
+def insert_window_event(
+    recording_timestamp: int, event_timestamp: int, event_data: dict
```

Comment:
Should `event_data` be:

```
event_data: dict[str, Any]
```

?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507